proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -0,0 +1,155 @@
+syntax = "proto3";
+
+package redpanda.api.console.v1alpha1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+
+// Request to start the creation of debug bundle process with given configuration parameters.
+message CreateDebugBundleRequest {
+  // Optional broker IDs. Do not set / leave empty to create for all.
+  repeated int32 broker_ids = 1;
+
+  // The size limit of the controller logs that can be stored in the bundle.
+  int32 controller_logs_size_limit_bytes = 2 [(buf.validate.field).int32 = {gte: 0}];
+
+  // For how long to collect samples for the CPU profiler
+  int32 cpu_profiler_wait_seconds = 3 [(buf.validate.field).int32 = {
+    gte: 15
+    lte: 600
+  }];
+
+  // Include logs dated from specified date onward.
+  google.protobuf.Timestamp logs_since = 4;
+
+  // Read the logs until the given size is reached.
+  int32 logs_size_limit_bytes = 5 [(buf.validate.field).int32 = {gte: 0}];
+
+  // Include logs older than the specified date.
+  google.protobuf.Timestamp logs_until = 6;
+
+  // Interval between metrics snapshots.
+  int32 metrics_interval_seconds = 7 [(buf.validate.field).int32 = {gte: 0}];
+
+  // Partition IDs. When provided, rpk saves extra admin API requests for those partitions.
+  // Optional.
+  repeated int32 partition_ids = 8;
```

Comment:
the partition option is of the format:
`{namespace/}topic/{partition ids}` where namespace is optional and will be replaced with `kafka` if not provided, and `parition ids` is comma separated numbers so:

`kafka/foo/1,2,3`. also there can be multiple of those so
`['kafka/foo/1,2,3', 'private/baz/3.4.5']`

Reply:
Hmm that's good to know... I was assuming it was just the integer partition IDs.
Changed to a repeated string value and added comments.

Reply:
Yeah sorry I named this field after the cli option from rpk debug bundle:

`  -p, --partition stringArray               Comma-separated partition IDs; when provided, rpk saves extra admin API requests for those partitions. Check help for extended usage`